src/containers/Tasks/index.tsx

```diff
@@ -3,15 +3,16 @@ import {useHistory, useParams} from 'react-router-dom';
 import formatDistanceToNow from 'date-fns/formatDistanceToNow';
 import parseISO from 'date-fns/parseISO';
 import intersection from 'lodash/intersection';
-
+import Icon, {IconType} from 'components/Icon';
 import {REPOSITORY_FILTERS} from 'constants/github';
 
 import {BreadcrumbMenu, EmptyPage, FlatNavLinks, LabelFilter, Loader, PageTitle} from 'components';
 import {GenericVoidFunction} from 'types/generic';
+import {SortBy} from 'types/tasks';
 import {Issue, Repository, RepositoryUrlParams} from 'types/github';
 import {fetchGithubIssues} from 'utils/github';
 import {sortByNumberKey} from 'utils/sort';
-
+import {DropdownInput} from 'components/DropdownInput';
```

Comment:
Alphabetical order for imports as well. Also refer [style guide](https://www.thenewboston.com/style-guide/react)